src/Evolution/Systems/Cce/OptionTags.hpp

```diff
@@ -204,8 +209,13 @@ struct ExtractionRadius : db::SimpleTag {
   using option_tags = tmpl::list<OptionTags::ExtractionRadius>;
 
   static constexpr bool pass_metavariables = false;
-  static double create_from_options(const double extraction_radius) noexcept {
-    return extraction_radius;
+  static double create_from_options(
+      const std::optional<double> extraction_radius) noexcept {
+    ASSERT(
+        static_cast<bool>(extraction_radius),
```

Comment:
we predominantly use the cast notation

Reply:
Didn't the cast notation arise because `boost::optional` doesn't have the equivalent of `has_value` and clang-tidy complained about the implicit cast?  I find `has_value` much easier to read and slightly more type-safe (e.g. if someone converts `std::optional<T>` to `T` they may miss the cast...).  Unless there is a compelling reason not to, I like the suggestion of using `has_value`

Reply:
I wasn't trying to suggest that we should use one over the other. I personally prefer keeping consistency throughout the `*::optional` (we should replace boost optional anyway) but will not object to either usage

Reply:
I agree we should be consistent, and replacing boost optional is beyond this PR